squirrel/PSPBUILD

```diff
@@ -9,35 +9,22 @@ license=('custom')
 depends=()
 makedepends=()
 optdepends=()
-source=(
-    "http://sourceforge.net/projects/squirrel/files/${pkgname}_3_0_7_stable.tar.gz"
-    "${pkgname}-${pkgver}.patch"
-)
-sha256sums=(
-    'c7c2548e2d2d74116303445118e197f585a3a5e6bde06fdfe668c05b1cb43fa2'
-    'SKIP'
-)
-
-prepare() {
-    cd "SQUIRREL3"
-    patch -p1 < ../${pkgname}-${pkgver}.patch
-}
+source=("git+https://github.com/albertodemichelis/squirrel.git#commit=c52b25d864400197fa4d64c46254d8b1de06c2fa")
```

Comment:
Please use the upstream tar.gz if possible

Reply:
It is not possible, the latest release was done in 2016, and more than 100 commits has been done, needed to be PSP able to compile